src/js/control-bar/progress-control/load-progress-bar.js

```diff
@@ -54,54 +77,53 @@ class LoadProgressBar extends Component {
    * @listens Player#progress
    */
   update(event) {
-    const liveTracker = this.player_.liveTracker;
-    const buffered = this.player_.buffered();
-    const duration = (liveTracker && liveTracker.isLive()) ? liveTracker.seekableEnd() : this.player_.duration();
-    const bufferedEnd = this.player_.bufferedEnd();
-    const children = this.partEls_;
-    const controlTextPercentage = this.$('.vjs-control-text-loaded-percentage');
-
-    // get the percent width of a time compared to the total end
-    const percentify = function(time, end, rounded) {
-      // no NaN
-      let percent = (time / end) || 0;
-
-      percent = (percent >= 1 ? 1 : percent) * 100;
-
-      if (rounded) {
-        percent = percent.toFixed(2);
+    this.requestAnimationFrame(() => {
```

Comment:
Without `requestAnimationFrame` this ran on the main thread, and it took a good chunk of time too.

Reply:
hiding whitespace changes makes this a lot easier to diff.